__tests__/lib/dev-environment/dev-environment-core.js

```diff
@@ -25,22 +25,28 @@ import { getEnvironmentPath,
 import { searchAndReplace } from '../../../src/lib/search-and-replace';
 import { resolvePath } from '../../../src/lib/dev-environment/dev-environment-cli';
 import { DEV_ENVIRONMENT_NOT_FOUND } from '../../../src/lib/constants/dev-environment';
+import { bootstrapLando } from '../../../src/lib/dev-environment/dev-environment-lando';
 
 jest.mock( 'xdg-basedir', () => ( {} ) );
-jest.mock( 'fs' );
 jest.mock( '../../../src/lib/api/app' );
 jest.mock( '../../../src/lib/search-and-replace' );
 jest.mock( '../../../src/lib/dev-environment/dev-environment-cli' );
 
 describe( 'lib/dev-environment/dev-environment-core', () => {
+	const cleanup = () => fs.rmSync( path.join( os.tmpdir(), 'lando' ), { recursive: true, force: true } );
+
+	beforeAll( cleanup );
+	afterAll( cleanup );
```

Comment:
When we have an existing `/tmp/lando` directory, this sometimes leads to interesting situations when tests pass locally and fail on CI.

Make sure we always start from a clean slate to be able to catch mocking bugs.